### ISSUE TITLE
[bug]: I can no longer generate images with any new SDXL models that I install.

### ISSUE THREAD

**Issue by StrikeLancer** [invoke-ai/InvokeAI#5494]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

12GB

### What version did you experience this issue on?

3.60

### What happened?

Since I upgraded to Invoke 3.60 I can no longer generate images with any new XDL models that I install. XDL models that I had installed with the previous version seem to work fine. If go into the Model Manager and click on the button "Convert To Diffusers" I get the same error.

### Screenshots

[2024-01-13 12:31:29,566]::[InvokeAI]::INFO --> Converting E:\InvokeAI\models\realWowXL_v2Baked.safetensors to diffusers format
[2024-01-13 12:31:29,702]::[InvokeAI]::ERROR --> Traceback (most recent call last):
  File "E:\InvokeAI\.venv\Lib\site-packages\invokeai\app\services\invocation_processor\invocation_processor_default.py", line 104, in __process
    outputs = invocation.invoke_internal(
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\InvokeAI\.venv\Lib\site-packages\invokeai\app\invocations\baseinvocation.py", line 669, in invoke_internal
    output = self.invoke(context)
             ^^^^^^^^^^^^^^^^^^^^
  File "E:\InvokeAI\.venv\Lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "E:\InvokeAI\.venv\Lib\site-packages\invokeai\app\invocations\compel.py", line 315, in invoke
    c1, c1_pooled, ec1 = self.run_clip_compel(
                         ^^^^^^^^^^^^^^^^^^^^^
  File "E:\InvokeAI\.venv\Lib\site-packages\invokeai\app\invocations\compel.py", line 172, in run_clip_compel
    tokenizer_info = context.services.model_manager.get_model(
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\InvokeAI\.venv\Lib\site-packages\invokeai\app\services\model_manager\model_manager_default.py", line 112, in get_model
    model_info = self.mgr.get_model(
                 ^^^^^^^^^^^^^^^^^^^
  File "E:\InvokeAI\.venv\Lib\site-packages\invokeai\backend\model_management\model_manager.py", line 490, in get_model
    model_path = model_class.convert_if_required(
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\InvokeAI\.venv\Lib\site-packages\invokeai\backend\model_management\models\sdxl.py", line 140, in convert_if_required
    return _convert_ckpt_and_cache(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\InvokeAI\.venv\Lib\site-packages\invokeai\backend\model_management\models\stable_diffusion.py", line 289, in _convert_ckpt_and_cache
    convert_ckpt_to_diffusers(
  File "E:\InvokeAI\.venv\Lib\site-packages\invokeai\backend\model_management\convert_ckpt_to_diffusers.py", line 1728, in convert_ckpt_to_diffusers
    pipe = download_from_original_stable_diffusion_ckpt(checkpoint_path, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\InvokeAI\.venv\Lib\site-packages\invokeai\backend\model_management\convert_ckpt_to_diffusers.py", line 1406, in download_from_original_stable_diffusion_ckpt
    set_module_tensor_to_device(unet, param_name, "cpu", value=param)
  File "E:\InvokeAI\.venv\Lib\site-packages\accelerate\utils\modeling.py", line 285, in set_module_tensor_to_device
    raise ValueError(
ValueError: Trying to set a tensor of shape torch.Size([640, 640]) in "weight" (which has shape torch.Size([640, 640, 1, 1])), this look incorrect.

[2024-01-13 12:31:29,719]::[InvokeAI]::ERROR --> Error while invoking:
Trying to set a tensor of shape torch.Size([640, 640]) in "weight" (which has shape torch.Size([640, 640, 1, 1])), this look incorrect.

**Comment by Millu** [comment]
Can you share an example of a model you're having issues with?

**Comment by StrikeLancer** [comment]
I have tried these new SDXL model installs:

Real_WoW_XL (V2 and V2 Baked)
https://civitai.com/models/185115/realwowxl

FULLY_REAL_XL
https://civitai.com/models/227059/fullyrealxl

Atrium V2.0 - TurboBoosted
https://civitai.com/models/216439?modelVersionId=291864

**Comment by StrikeLancer** [comment]
I am using the Model Manager>Import Models>Advanced

I reinstalled Invoke from scratch, just to be sure. I then added one of the SDXL models into the autoimport folder and the model started converting and working. I looked at the installed configuration for the model and I now believe that when I was installing them myself it was using  v1-inference.yaml and not sd_xl_base.yaml.

In the previous version I was using the "Scan For Models" option to install them, so I had no issues.

**Comment by StrikeLancer** [comment]
I deleted my last reply. I started getting black renders. I reinstalled everything from scratch again and it's now working for me as it should, as far as my original issue. I restarted my computer several times in the last 10 hours to make sure everything was working between restarts with no issues.

This issue can be closed for me. I really can't explain what I did, except to say I reinstalled, a few times. That's it.

I installed Automatic1111 for the first time yesterday and generated images with several SD and SDXL models with it, amid my issues with Invoke 3.60, with no issues, so it seems it wasn't something I was doing wrong. I kept trying with Invoke because the Automtic1111 UI doesn't fit me. I am happy that I stuck with it.

Thanks.
